cmd/corectl/no_mockhsm.go

```diff
@@ -4,6 +4,6 @@ package main
 
 import "chain/database/sql"
 
-func createBlockKeyPair(db *sql.DB, args []string) {
-	fatalln("error: create-block-keypair disabled in prod build")
+func createBlockKeyPair(*sql.DB, []string) {
+	fatalln("error: create-block-keypair disabled in this build")
```

Comment:
Now that this is protected by cored, I think we can get rid of corectl's no_mockhsm build tag.